src/main/scala/Column.scala

```diff
@@ -404,6 +405,16 @@ case class Column private (
     warnings: Array[ErrorWithCsvContext],
     var errors: Array[ErrorWithCsvContext]
 ) {
+  val minLength: Option[Int] =
```

Comment:
Wouldn't it make more sense to have this in the `fromJson` method and not need to pass datatype through at all?

Reply:
I have moved them to fromJson function, but still passing the datatype jsonNode since I am using it in an error.